Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -112,20 +112,19 @@ public class InstallationManager : IInstallationManager
         /// <inheritdoc />
         public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationToken cancellationToken = default)
         {
-            using (var response = await _httpClient.SendAsync(
+            using var response = await _httpClient.SendAsync(
                 new HttpRequestOptions
                 {
                     Url = "https://repo.jellyfin.org/releases/plugin/manifest.json",
                     CancellationToken = cancellationToken,
                     CacheMode = CacheMode.Unconditional,
                     CacheLength = TimeSpan.FromMinutes(3)
                 },
-                HttpMethod.Get).ConfigureAwait(false))
-            using (Stream stream = response.Content)
-            {
-                return await _jsonSerializer.DeserializeFromStreamAsync<IReadOnlyList<PackageInfo>>(
-                    stream).ConfigureAwait(false);
-            }
+                HttpMethod.Get).ConfigureAwait(false);
+            await using Stream stream = response.Content;
```

Comment:
When using `await using` you need to make sure to call `ConfigureAwait()`. See https://github.com/jellyfin/jellyfin/pull/2837